demo-projects/blog/index.js

```diff
@@ -112,8 +112,24 @@ keystone.createList('Comment', {
 
 const admin = new AdminUI(keystone, {
   adminPath: '/admin',
-  sortListsAlphabetically: true,
   authStrategy,
+  pages: [
+    {
+      label: 'Blog',
+      children: [
+        { listKey: 'Post' },
+        { label: 'Categories', listKey: 'PostCategory' },
+        {
+          label: 'Nested',
+          children: [{ label: 'Comments', listKey: 'Comment' }],
```

Comment:
I think the demo projects should show real world use cases so maybe you could remove the nested label and then add this to the basic test project?

Reply:
Mind if this comes later? This branch keeps going stale